ui/app/templates/components/two-step-button.hbs

```diff
@@ -32,7 +32,7 @@
   />
   <Hds::Button
     data-test-confirm-button
-    @size="medium"
+    @size={{@size}}
```

Comment:
Side effect: our two-step-buttons were always medium sized. This gives them all the flexibility of our other Helios buttons.